app/assets/stylesheets/application.scss

```diff
@@ -13,29 +13,46 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
- *= depend_on_asset "Landing_Page.jpg"
  */
 
  body {
-  background-image: url("<%= asset_path('Landing_Page.jpg') %>");
+  background-image: image_url("Landing_Page.jpg");
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores